src/main/java/exceptions/TodoItemError.java

```diff
@@ -0,0 +1,5 @@
+package exceptions;
+
+public class TodoItemError extends Exception {
```

Comment:
Error usually should inherit from RuntimeError class. Your inheritance is correct here, naming is incorrect.